Configor:
  DataStream:
    # key: event camera topic, value: event type. Supported event types:
    # 1. PROPHESEE_EVENT: https://github.com/prophesee-ai/prophesee_ros_wrapper.git
    # 2. DVS_EVENT: https://github.com/uzh-rpg/rpg_dvs_ros.git
    EventTopics:
      - key: "/dvs/events"
        value: "DVS_EVENT"
    BagPath: "/home/csl/dataset/event_develop/data_2024_12_01.bag"
    # the time piece: [BegTime, BegTime + Duration], unit: second(s)
    # if you want to use all time data for calibration, please set them to negative numbers
    # Note that the 'BegTime' here is measured from the start time of bag
    # and is not related to the timestamp of the data recorded in the package.
    BeginTime: 17
    Duration: 10
    OutputPath: "/home/csl/ros_ws/eKalibr/src/ekalibr/data/test"
  Preference:
    # supported data output format:
    # 1. JSON
    # 2. XML
    # 3. YAML
    # 4. BINARY (not recommended)
    OutputDataFormat: "YAML"
